semcore/mini-chart/src/component/score/donut.shadow.css

```diff
@@ -0,0 +1,17 @@
+@import '../skeleton/skeleton.shadow.css';
+
+SDonutContainer[semi] {
+  g {
+    transform: rotate(180deg) translate(-24px, -24px);
+  }
+}
+
+SDonutContainer {
+  g {
+    transform: rotate(-90deg) translate(-24px)
+  }
+}
+
+SDonutContainer[animate] circle {
+  transition: all 500ms ease-in-out;
```

Comment:
Probably it's better to use `--intergalactic-duration-extra-slow` token here (or create a new one for charts)

Reply:
added here and in another places